### PR TITLE
docs: add example of multiple databases for YAML

### PR DESCRIPTION
**Pull request by unknwon**

### CODE REVIEW

File: content/docs/tenant-database-management/tenant-gitops.md [hunk 1]
```diff
@@ -62,3 +62,15 @@ bytebase/Staging/store##0002##ddl##add-payout-table.sql
...
+
+To target multiple databases, add more entries to the `databases` section:
+
+```yml
+databases:
+  - name: supermarket
+  - name: supermarket_east
+  - name: supermarket_west
+statement: |
+  INSERT INTO company (id, name, address) VALUES (1, 'Bytebase', '1 DevOps street');
```
Comment (tianzhou) [inline content/docs/tenant-database-management/tenant-gitops.md:74]: LOL

File: content/docs/tenant-database-management/tenant-gitops.md [hunk 2]
```diff
@@ -29,12 +29,12 @@ Where `bytebase/` is the **Base directory**, depending on [your configuration](/
 <hint-block type="warning">
 
 1. This feature is currently experimental, behaviors are subject to change, please only use with guidance from our technical support.
-1. The **File path template** must use `.sql` as the file extension for migration scripts.
+1. The **File path template** must use `.sql` as the file extension for migration scripts, both SQL files and YAML manifest share the same setting of the **File path template**.
```
Comment (tianzhou) [inline content/docs/tenant-database-management/tenant-gitops.md:32]: Is `.sql` extension intended?

Comment (unknwon) [reply content/docs/tenant-database-management/tenant-gitops.md:32]: For the current implementation, yes. But we can lax to any file extension if we want. Just happen to choose the `.sql` to start with (as this is the most common file extension for SQL scripts).

**Review by candy2255 — APPROVED** [review]
LGTM

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**